pkg/kubectl/describe.go

```diff
@@ -311,6 +311,7 @@ func (d *PersistentVolumeDescriber) Describe(namespace, name string) (string, er
 		} else {
 			fmt.Fprintf(out, "Claim:\t%s\n", "")
 		}
+		fmt.Fprintf(out, "Reclamation Policy:\t%d\n", pv.Spec.PersistentVolumeReclaimPolicy)
```

Comment:
"Reclaim Policy" ? I know the grammar is a bit wierd, but reclamation is a hard word, and not any more helpful